### ISSUE TITLE
用demo设置为黑色之类的颜色，也是会显示下白色，在不同的设备都会出现，下面有代码，望楼主能回复下，感谢

### ISSUE THREAD

**Issue by swlizhao** [tigerAndBull/TABAnimated#183]
### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

#  操作步骤
  [TABAnimated sharedAnimated].animationType = TABAnimationTypeOnlySkeleton;
        int h = (int)((SCREEN_WIDTH - 30) * 0.475);
        _tableView.tabAnimated = [TABTableAnimated animatedWithCellClass:[LiveHallSkeletonCell class] cellHeight:h];
    
       _tableView.tabAnimated.adjustBlock = ^(TABComponentManager * _Nonnull manager) {
          manager.animation(0).color(RGB(28, 28, 28));
        };
        _tableView.tabAnimated.animatedBackgroundColor = [UIColor blackColor];
        _tableView.tabAnimated.animatedColor = RGB(44, 44, 44);
        _tableView.tabAnimated.animatedCount = 8;

**Comment by tigerAndBull** [comment]
自己看一下图层，描述的完全不明所以

**Comment by swlizhao** [comment]
问题已解决！  我的场景是控制器里面套控制器， 没有标记嵌套视图属性导致
